altsrc/flag_test.go

```diff
@@ -3,12 +3,12 @@ package altsrc
 import (
 	"flag"
 	"fmt"
+	"github.com/urfave/cli/v2"
```

Comment:
Meta comment - I'm ignoring the diff on altsrc since 🤷‍♀ who even really knows what's going on with altsrc